collect_app/src/androidTest/java/org/odk/collect/android/support/pages/FormManagementPage.java

```diff
@@ -19,4 +19,9 @@ public FormManagementPage assertOnPage() {
         clickOnString(R.string.form_update_frequency_title);
         return new ListPreferenceDialog<>(R.string.form_update_frequency_title, this).assertOnPage();
     }
+
+    public ListPreferenceDialog<FormManagementPage> clickAutoSend() {
```

Comment:
I was yes, but ended up not. Do you think it is still worthwhile to keep to use for testing in the future? Or get rid of it since we got rid of the `MainMenuPage#enableAutoSend` test?

Reply:
Good question. I guess it might come in handy some day! Given it's test code not app code it's not going to contribute to the binary size or anything so nice to have around.